tests/chainerx_tests/op_utils.py

```diff
@@ -79,6 +80,27 @@ def forward_chainerx(self, inputs):
         raise NotImplementedError(
             'Op test implementation must override `forward_chainerx`.')
 
+    def run_test_forward(self, backend_config):
+        # Skipping Forward -> Test Skipped
+        if self.skip_forward_test:
+            raise unittest.SkipTest('skip_forward_test is set')
+
+        super(OpTest, self).run_test_forward(backend_config)
+
+    def run_test_backward(self, backend_config):
+        # Skipping Backward -> Test PASS
+        if self.skip_backward_test:
+            return
+
+        super(OpTest, self).run_test_backward(backend_config)
+
+    def run_test_double_backward(self, backend_config):
+        # Skipping Double Backward -> Test PASS
+        if self.skip_double_backward_test:
+            return
+
+        super(OpTest, self).run_test_backward(backend_config)
```

Comment:
Ops, could you fix this?
```suggestion
        super(OpTest, self).run_test_double_backward(backend_config)
```

Reply:
Haha. Sorry. 

Thank You.